db/migrate/20170626162941_create_users.rb

```diff
@@ -1,7 +1,19 @@
 class CreateUsers < ActiveRecord::Migration[5.1]
   def change
     create_table :users do |t|
-
+	t.string :fname
```

Comment:
Since you're going to log every message that comes in, maybe `access_log` or `interaction`? Just a couple suggestions...